Update InputEvents.cpp for Skyrim Souls RE

derickso opened the pull request.

derickso committed:
Update InputEvents.cpp for Skyrim Souls RE

max-su-2019 reviewed (commented):
Pls use the [INPUT_CONTEXT_ID::kGameplay](https://github.com/max-su-2019/MaxsuDetectionMeter/blob/f7cdda4b8af72c7763c48d2ced631c1e3d399fad/src/Renderer.cpp#LL299C3-L299C3) there instead.

derickso committed:
Update InputEvents.cpp

INPUT_CONTEXT_ID

derickso commented:
Like that?

max-su-2019 commented:
Should be 
```cpp
ctrlMap->contextPriorityStack.back() != RE::UserEvents::INPUT_CONTEXT_ID::kGameplay
```

derickso committed:
Update InputEvents.cpp

Update input context logic

"You only perform dodge when the current control context is gameplay mode which means your input would moving the player character."
